test/test_command_line_interface.py

```diff
@@ -42,17 +42,17 @@ def _test(self, args, expect):
             with mock_stream() as stdout:
                 cli.parse_args(args)
         text = stdout.getvalue()
-        self.assertTrue(text.startswith(expect))
+        self.assertRegex(text, expect)
 
     def test_global_help(self):
-        self._test(['-h'], 'usage: TESTSUITE [-h]')
+        self._test(['-h'], r'^usage: TESTSUITE\s+\[-h\]')
```

Comment:
Would this be acceptable?
```suggestion
        self._test(['-h'], r'^usage: TESTSUITE \[-h\]')
```
It would provide a way of seeing what the difference between the two strings is.

Reply:
Yes, that would be acceptable and the argument to see the diff when the test fails is good. Hopefully your sandbox in #263 is *not* fixed by this becaue I think it is another issue and I would not want to fix it with some random edit.

I think with this we can merge it.

Can we unlink #263 somehow? I don't know if you want to keep it open.

Reply:
It does not fix #263 but it makes it easier to add a patch that does to nixpkgs.